src/Files.App/Views/Settings/FoldersPage.xaml

```diff
@@ -136,11 +136,20 @@
 						<local:SettingsBlockControl Title="{helpers:ResourceString Name=GroupInDescendingOrder}" HorizontalAlignment="Stretch">
 							<ToggleSwitch
 								AutomationProperties.Name="{helpers:ResourceString Name=GroupInDescendingOrder}"
-								IsEnabled="{x:Bind ViewModel.isDefaultGrouped, Mode=OneWay}"
+								IsEnabled="{x:Bind ViewModel.IsDefaultGrouped, Mode=OneWay}"
 								IsOn="{x:Bind ViewModel.GroupInDescendingOrder, Mode=TwoWay}"
 								Style="{StaticResource RightAlignedToggleSwitchStyle}" />
 						</local:SettingsBlockControl>
 
+						<!--  Group by Month  -->
```

Comment:
```suggestion
						<!--  Group By Month  -->
```

Reply:
I don't think short prepositions are generally capitalized in titles.